components/commands/common/accelerator_pref_manager.h

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) 2023 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at https://mozilla.org/MPL/2.0/.
+
+#ifndef BRAVE_COMPONENTS_COMMANDS_COMMON_ACCELERATOR_PREF_MANAGER_H_
+#define BRAVE_COMPONENTS_COMMANDS_COMMON_ACCELERATOR_PREF_MANAGER_H_
+
+#include <vector>
+
+#include "base/component_export.h"
+#include "base/containers/flat_map.h"
+#include "components/prefs/pref_registry_simple.h"
+#include "components/prefs/pref_service.h"
+#include "ui/base/accelerators/accelerator.h"
+
+namespace commands {
+
+class COMPONENT_EXPORT(COMMANDS_COMMON) AcceleratorPrefManager {
```

Comment:
Cool, thanks for that explanation it's really helpful. I've removed the `//chrome` and `//content` deps, I was cargo culting and existing test but it's significantly simpler using `TestingPrefServiceSimple`.

Thanks @goodov!

Reply:
> Cool, thanks for that explanation it's really helpful. I've removed the `//chrome` and `//content` deps, I was cargo culting and existing test but it's significantly simpler using `TestingPrefServiceSimple`.
> 
> Thanks @goodov!

Sorry, I might've been not quite clear here, but you should've moved all these parts from `components/commands/common` to `components/commands/browser`, no need to move everything to `//brave/browser`.

> I've removed the //chrome and //content deps

that's a good thing.

Reply:
I've got another branch testing moving `accelerator_parsing.h` back to `components/commands` which I think should work (looks like a POSIX include had snuck in) https://github.com/brave/brave-core/pull/17420

After https://github.com/brave/brave-core/pull/17406 lands it should be okay to move `accelerator_service.h` to `components/commands/browser`, as it will remove the dependency on `brave/app` & `chrome/app` (it generates a file using integers rather than the defines in those files).